src/mono/wasm/build/WasmApp.targets

```diff
@@ -259,8 +259,12 @@
      <_WasmRuntimePackNativeLibs Include="libSystem.Native.a"/>
      <_WasmRuntimePackNativeLibs Include="libSystem.IO.Compression.Native.a"/>
      <_WasmRuntimePackNativeLibs Include="libmono-profiler-aot.a"/>
+     <!-- these can't conditional until we've solved the linking issue
```

Comment:
It would be nice to put a link to what "the linking issue" is here, so if someone reads this they know what it is talking about.

Reply:
I verified the code is linked out when the build is correct so forcing them to be conditional here is only an optimization.  #49499 is the issue tracking what might be breaking the build in CI.